### DISCUSSION TITLE
Main difference between the README contents and the website contents?

### DISCUSSION THREAD

**Discussion by ludgie** [labwc/labwc#1440] [Ideas]
I think by having a main and unique place where all the information for the user is available (in this case labwc.github.io) would be great as it'd allow people to discover better the project.
Thus, maybe implementing some contents of the README file onto the website would be interesting

I suggest the following to do:
- Add scope and IRC channel links under the website
- Add dedicated page to see NEWS.md (though maybe unnecessary)
- Copy project description to a new page on the website, so people can now what the project is about more deeply
- Add translations info to the website
- Maybe cleanup the README file and redirect to website?

Though here it arises the question whether the README should be like the information file for "advanced" users/contributors while the website should be more of the presentation, thus not needing some of the technical information from the README

**Comment by ahesford** [comment]
In addition to holding documentation in user-consumable form, the web site already contains a project description and a link to the README. "Discovery" isn't really enhance by moving content from one existing site that people may or may or may not find to another existing site that people may or may not find.

I fail to see the benefit of moving more of the README content to the web site. If the content is simply duplicated there, it adds the burden of reconciling changes; the current hyperlink eliminates that burden. Moving, rather than duplicating, the README content to the site is bad, because the README is integral to the project repository and provides information from the repository without the need to consult another site (or even open a browser, if you're looking for information from a clone or archive of the repository).

**Reply by ludgie** [reply]
Got it, thank you
